### ISSUE TITLE
Documentation has a typo in Bloom filter territory

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/leveldb#105]
```

Here:
 http://leveldb.googlecode.com/svn/trunk/doc/index.html

Please replace NewBloomFilter with NewBloomFilterPolicy to match 
filter_policy.h.

Inside the plex we would suggest:

    s/NewBloomFilter/NewBloomFilterPolicy/g

thx,
 Dave

PS
 I can attach a diff or 'patch' if you want - plz advise.
```

---

Original issue reported on code.google.com by `dspen...@google.com` on 21 Jul 2012 at 5:20

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.7.0.

```

---

Original comment by `san...@google.com` on 16 Oct 2012 at 11:30

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `san...@google.com` on 16 Oct 2012 at 11:30
- Changed state: **Fixed**
